charts/aws-otel-collector/templates/configmap.yaml

```diff
@@ -37,6 +37,11 @@ data:
             metric_name_selectors:
               - pod_cpu_utilization
               - pod_memory_utilization
+          - dimensions: [[ClusterName]]
+            metric_name_selectors:
+              - cluster_node_count
+              - node_cpu_utilization
+              - node_memory_utilization
```

Comment:
Looks good but could we also add the `replicas_ready` metric for use as it was the catalyst for the original ticket https://snplow.atlassian.net/browse/TECOPS-11383 

It's only available in CW Observability but would be good to have as support / engineering are using this to track number of replicas in grafana